src/VueProgram.ts

```diff
@@ -0,0 +1,131 @@
+import fs = require('fs');
+import path = require('path');
+import ts = require('typescript');
+import FilesRegister = require('./FilesRegister');
+import FilesWatcher = require('./FilesWatcher');
+import vueParser = require('vue-parser');
+
+class VueProgram {
+  static loadProgramConfig(configFile: string) {
+    const extraExtensions = ['vue'];
+
+    const parseConfigHost: ts.ParseConfigHost = {
+        fileExists: ts.sys.fileExists,
+        readFile: ts.sys.readFile,
+        useCaseSensitiveFileNames: ts.sys.useCaseSensitiveFileNames,
+        readDirectory: (rootDir, extensions, excludes, includes, depth) => {
+            return ts.sys.readDirectory(rootDir, extensions.concat(extraExtensions), excludes, includes, depth);
+        }
+    };
+
+    const parsed = ts.parseJsonConfigFileContent(
+      // Regardless of the setting in the tsconfig.json we want isolatedModules to be false
+      Object.assign(ts.readConfigFile(configFile, ts.sys.readFile).config, { isolatedModules: false }),
+      parseConfigHost,
+      path.dirname(configFile)
+    );
+
+    parsed.options.allowNonTsExtensions = true;
+
+    return parsed;
+  }
+
+  /**
+   * Since 99.9% of Vue projects use the wildcard '@/*', we only search for that in tsconfig CompilerOptions.paths.
```

Comment:
Please, add information about vue support and how it resolves "@/*" paths in the `README.md` :)

Reply:
Added `vue section` to README. :smile: